Dungeoneer/src/com/interrupt/dungeoneer/overlays/LevelUpOverlay.java

```diff
@@ -71,7 +72,9 @@ protected void addAttribute(Table table, final String text, final Integer curren
 		attributeName.setWrap(true);
 		attributeName.setColor(new Color(0.298039215686275f, 0.12156862745098f, 0.12156862745098f, 1f));
 
-		final Label value = new Label("Current: " + currentValue.toString(), skin.get(LabelStyle.class));
+		String currentStatValueLabel = StringManager.get("overlays.LevelUpOverlay.currentStatValueLabel");
+		currentStatValueLabel = MessageFormat.format(currentStatValueLabel, currentValue,toString());
```

Comment:
Did you mean `currentValue.toString()`?

Reply:
Yes, yes I did. 🤦 
Though I might not need it?

Reply:
Puh, I don't know? From the usage I think `currentValue` comes in as an integer, right? And maybe the `toString` method is called automatically?

However, the level up screen is nothing that is called every frame and, therefore, I guess, better safe than sorry :D

Reply:
`MessageFormat.format()` takes a string and an arbitrary number of Objects as params, so I've just dropped `,toString()`